packages/clipboard/src/index.ts

```diff
@@ -26,11 +26,11 @@ declare module "solid-js" {
  *
  * @example
  * ```ts
- * const [ write, read ] = createClipboard();
+ * const [ write, read ] = makeClipboard();
  * console.log(await read());
  * ```
  */
-const createClipboard = (): [
+export const makeClipboard = (): [
   write: ClipboardSetter,
   read: () => Promise<string>,
```

Comment:
Should we take this opportunity to switch [write, read, something_else] to [read, write, something_else]?
Maybe writing is more common operation, but [read, write] is a more common api.

Reply:
Yup I'll be sure to do that, good point.